public/splash.html

```diff
@@ -0,0 +1,138 @@
+<!-- Splash screen built upon vanilla html, js and css -->
+<html>
+  <head>
+    <script defer type="text/javascript">
+      // defer is for executing js before the page shows up
+      const queryString = window.location.search;
+      const urlParams = new URLSearchParams(queryString);
+
+      const primary = urlParams.get("primary");
+      const background = urlParams.get("background");
+      const display = urlParams.get("display");
+      const version = urlParams.get("version");
+
+      const root = document.documentElement;
+
+      // Notice: when passing url parameters, `#` is rstriped for unicode issues
+      root.style.setProperty("--primary", `#${primary}`);
+      root.style.setProperty("--background", `#${background}`);
+      root.style.setProperty("--display", `#${display}`);
+
+      window.onload = function () {
+        document.getElementById("version").innerHTML = `version ${version}`;
+      };
+    </script>
+    <style>
```

Comment:
```suggestion
    <style>
      @font-face {
        font-family: "Rounded M+ 1p";
        src: url("../src/fonts/unhinted-rounded-mplus-1p-regular.woff2");

        font-weight: normal;
      }

      @font-face {
        font-family: "Rounded M+ 1p";
        src: url("../src/fonts/unhinted-rounded-mplus-1p-bold.woff2");

        font-weight: bold;
      }
```

I suggest using "Rounded M+ 1p" as the title font.
Since #1061, "Rounded M+ 1p" is now used as the default font.
"Rounded M+ 1p" is VOICEVOX logo font.
I think it would be more "オシャレ" to unify the font of the splash screen with the logo font.